scripts/devaccess/helper_scripts/3scale-dump.sh

```diff
@@ -0,0 +1,1314 @@
+#!/bin/bash
+
+THREESCALE_PROJECT="${1}"
+
+COMPRESS_UTIL="${2,,}"
+
+#CURRENT_DIR=$(dirname "$0")
+CURRENT_DIR=$PWD
+
+# Avoid fetching information about any pod that is not a 3scale one
+THREEESCALE_PODS=("3scale-operator" "apicast-production" "apicast-staging" "apicast-wildcard-router" "backend-cron" "backend-listener" "backend-redis" "backend-worker" "system-app" "system-memcache" "system-mysql" "system-redis" "system-resque" "system-sidekiq" "system-sphinx" "zync" "zync-que" "zync-database")
+
+NOW=$(date +"%Y-%m-%d_%H-%M" -u)
+
+DUMP_DIR="${CURRENT_DIR}/3scale_dump-${NOW}"
+
+# using hardcoded filename is safe now since all output files are now inside a new unique temp dir
+DUMP_FILE="3scale-dump.tar"
+
+
+REDACT_SCRIPT="redact_env_vars.py"
```

Comment:
To avoid any issues about whether or not this script is in $PATH, make the path explicit.

```suggestion
REDACT_SCRIPT="/usr/local/bin/redact_env_vars.py"
```